pkg/builders/s2i/builder_test.go

```diff
@@ -500,6 +500,6 @@ func (n notFoundErr) Error() string {
 	return "not found"
 }
 
-func (n notFoundErr) NotFound() bool {
-	return true
+func (n notFoundErr) NotFound() {
+	panic("just a marker interface")
```

Comment:
Just a note: this change was tricky, it was not forced by compiler error like rest.
There was just failing test since the mock error stopped being "not found" error according to docker semantic.

Reply:
For future I added type assert below.